dot_lpdev_cmds.sh

```diff
@@ -450,13 +535,20 @@ function __lpdev_node_reset {
   transcoderRunning=false
   unset transcoderGeth
 
+  unset verifierPid
+  verifierRunning=false
+  unset verifierGeth
+
+  unset verifierIPFSPid
+  verifierIPFSRunning=false
 }
 
 function __lpdev_node_update {
 
   wget_args=$1
 
-  URL=$(curl -s https://api.github.com/repos/livepeer/go-livepeer/releases |jq -r ".[0].assets[].browser_download_url" | grep linux)
+  # URL=$(curl -s https://api.github.com/repos/livepeer/go-livepeer/releases |jq -r ".[0].assets[].browser_download_url" | grep linux)
+  URL=https://github.com/livepeer/go-livepeer/releases/download/0.1.15/livepeer_linux.tar
```

Comment:
We should switch back to using the github `releases` endpoint so we can always get the latest release

Reply:
Fixed in a commit update